molecule/resources/verify.yml

```diff
@@ -64,3 +64,22 @@
         - name: Show OpenWisp log
           debug:
             var: openwisp_log
+
+    - name: Check Freeradius
+    # TODO: This test should work when openwisp-radius 0.3.0 is released!
```

Comment:
This remains a blocker to merge this PR.
We would need to release the next version of openwisp-radius to enable testing here.

Reply:
yes we're getting near to that :+1: